src/qmapshack/icon/CMapIconSizesSetup.h

```diff
@@ -0,0 +1,43 @@
+/**********************************************************************************************
```

Comment:
Either everything in "helpers" or all dialogs and stuff  concerning icons in "icons"